.github/workflows/call-build-windows.yaml

```diff
@@ -112,6 +112,27 @@ jobs:
         with:
           arch: ${{ matrix.config.arch }}
 
+      - name: Get gzip command w/ chocolatey
+        uses: crazy-max/ghaction-chocolatey@v2
+        with:
+          args: install gzip -y
+
+      # http://man7.org/linux/man-pages/man1/date.1.html
+      - name: Get Date
+        id: get-date
+        run: |
+          echo "date=$(/bin/date -u "+%Y%m%d")" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - name: Restore cached packages of vcpkg
+        id: cache-vcpkg-sources
+        uses: actions/cache/restore@v3
+        with:
+          path: |
+            C:\vcpkg\packages
+          key: ${{ runner.os }}-${{ matrix.config.arch }}-vcpkg
+          enableCrossOsArchive: false
+
       - name: Build openssl with vcpkg
```

Comment:
Do we need to skip on cache hit?

Reply:
No. We're only caching for sources of openssl and libyaml. We always need to build them on the every run.